packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -10,7 +10,6 @@ import { useRtl } from '@mui/system/RtlProvider';
 import { styled, createUseThemeProps } from '../zero-styled';
 import useTheme from '../styles/useTheme';
 import debounce from '../utils/debounce';
-import { getNormalizedScrollLeft, detectScrollType } from '../utils/scrollLeft';
```

Comment:
These are no longer needed as `negative` has been standardized on all supported browsers: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollLeft#browser_compatibility

Tested in Chrome, Safari, Firefox, and Edge. Recording from Edge:


https://github.com/mui/material-ui/assets/16889233/4450b406-49ba-426f-9841-190962f72f39

Reply:
Oh wow, they did, it has never been either to build great components, fewer weird browser bugs to fix 😄.